server/execution/entities/player.py

```diff
@@ -43,3 +45,6 @@ def to_json(self, shallow: bool = False):
         only the object reference in form of a unique identifier is included.
         """
         return json.dumps(self.to_dict(shallow))
+
+    def has_logged_in(self) -> bool:
```

Comment:
Useless method. Python allows directly access for such a request. Any further implementation is located in the endpoint method